spec/spec.md

```diff
@@ -278,7 +279,7 @@ After the [session establishment](#session-establishment) process completes, eve
 
 After decrypting the ciphertext with the shared secret generated during [session establishment](#session-establishment) and verifying the authentication tag, the receiving endpoint should further interpret it as a [JSON-RPC 2.0](https://www.jsonrpc.org/specification) message.
 
-The message sequence number is monotonically increasing, and starts at 1 when session establishment completes. Each endpoint maintains its own independent sequence number, and increments it each time an encrypted message is created and sent. On receipt of an encrypted message, each endpoint should ensure that the sequence number is greater than that of the previous message received (other than for the first message received). If the value is less than or equal to the previously observed sequence number, the encrypted message should be discarded and the connection closed.
+The message sequence number is monotonically increasing, and starts at 1 when session establishment completes. Each endpoint maintains its own independent sequence number, and increments it by 1 each time an encrypted message is created and sent. On receipt of an encrypted message, each endpoint should verify that the sequence number is 1 greater than that of the previous message received (other than for the first message received). On receipt of a message with a sequence number set to anything other than the expected next value, the encrypted message should be discarded and the connection closed.
```

Comment:
I remember having a conversation about this, and the conclusion of that conversation being that it was OK for a message to be missed (ie. for the sequence number to be more than one greater than the last observed sequence number.

In particular, our code today doesn't do what this spec change would demand.

https://github.com/solana-mobile/mobile-wallet-adapter/blob/39301c7c57d19ed5636aeacae4181b447d236cc2/js/packages/mobile-wallet-adapter-protocol/src/transact.ts#L150-L152

Thoughts?